docs/guides/README.md

```diff
@@ -4,8 +4,8 @@ sort: 21
 
 # Guides
 
-1. [K8s monitoring via VM Single](k8s-monitoring-via-vm-single.html)
-2. [K8s monitoring via VM Cluster](k8s-monitoring-via-vm-cluster.html)
-3. [HA monitoring setup in K8s via VM Cluster](k8s-ha-monitoring-via-vm-cluster.html)
-4. [Getting started with VM Operator](getting-started-with-vm-operator.html)
-5. [Multi Retention Setup within VictoriaMetrics Cluster](guide-vmcluster-multiple-retention-setup.html)
\ No newline at end of file
+1. [K8s monitoring via VM Single](k8s-monitoring-via-vm-single.md)
```

Comment:
why `.md`? https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single.html looks fine

Reply:
> why `.md`? https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single.html looks fine

is https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single.html?
Because in docs/guides/README.md, the url is https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/guides/k8s-monitoring-via-vm-single.html , the url is 404, so I changed it to .md

Reply:
> why `.md`? https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single.html looks fine

If the url is https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single.html, let me modify it.

Reply:
> Because in docs/guides/README.md, the url is https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/guides/k8s-monitoring-via-vm-single.html , the url is 404, so I changed it to .md

I see. It is a relative path for all links in `docs/guides/README.md`, so that's why it adds all this `/blob/master...` when looking at page https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/guides/README.md.
However, all docs placed under the `docs` folder are used for building https://docs.victoriametrics.com/ website and all links at https://docs.victoriametrics.com/guides/ are fine (because they're relative).
It is better to update those links to the absolute path like `https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single.html`.

Reply:
> > Because in docs/guides/README.md, the url is https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/guides/k8s-monitoring-via-vm-single.html , the url is 404, so I changed it to .md
> 
> I see. It is a relative path for all links in `docs/guides/README.md`, so that's why it adds all this `/blob/master...` when looking at page https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/guides/README.md. However, all docs placed under the `docs` folder are used for building https://docs.victoriametrics.com/ website and all links at https://docs.victoriametrics.com/guides/ are fine (because they're relative). It is better to update those links to the absolute path like `https://docs.victoriametrics.com/guides/k8s-monitoring-via-vm-single.html`.

Yes, that's right.
i updated.